File: poo/Java/DivisaoExata/DivisaoExata.java
import java.util.Scanner;

public class DivisaoExata {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        long x = scanner.nextLong();
        long y = scanner.nextLong();
        if (y != 0 && x % y == 0) {
            System.out.println("a divisao de " + x + " por " + y + " eh exata.");
        }
        scanner.close();
    }
}

File: poo/Cpp/DivisaoExata/DivisaoExata.cpp
#include <iostream>

int main() {
    long long x, y;
    std::cin >> x >> y;
    if (x % y == 0) {
        std::cout << "a divisao de " << x << " por " << y << " eh exata." << std::endl;
    }
    return 0;
}